src/server/routes/error/error.ts

```diff
@@ -0,0 +1,10 @@
+import { Router, Request, Response } from 'express';
+
+var router = Router();
+
+router.post('/', (req: Request, res: Response) => {
+  console.error(`Client Error: ${JSON.stringify(req.body)}`);
```

Comment:
could you validate that `req.body` at least has a string `message` it is just so strange to see a route with no validation at all... feels so naked